facefusion/processors/modules/face_editor.py

```diff
@@ -15,7 +15,7 @@
 from facefusion.execution import create_inference_pool
 from facefusion.face_analyser import get_many_faces, get_one_face
 from facefusion.face_helper import paste_back, warp_face_by_face_landmark_5
-from facefusion.face_masker import create_face_mask, create_occlusion_mask, create_static_box_mask
+from facefusion.face_masker import create_occlusion_mask, create_static_box_mask
```

Comment:
can we delete create_face_mask then?

Reply:
It is only needed for expression-restorer. so far face-editor doesn't needs it.